### PR TITLE
docs/Add release instructions

### PR DESCRIPTION
**Pull request by carpawell**

### CODE REVIEW

File: docs/release-instruction.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Add an entry to the `CHANGELOG.md` following the style established there. Add an
+optional codename(for not patch releases), version and release date in the heading.
+Write a paragraph describing the most significant changes done in this release. Add
+`Fixed`, `Added`, `Removed` and `Updated` sections with fixed bug, new features and
+other changes.
+
+Open Pull Request (must receive at least one approval) and merge this changes.
+
+## Make a Github release
```
Comment (cthulhu-rider) [inline docs/release-instruction.md:20]: Nothing about tag?

File: docs/release-instruction.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Using Github's web interface create a new release with the same changes from changelog
+and publish it.
+
+## Close github milestone
+
+Close corresponding vX.Y.Z github milestone.
+
+## Post-release
+
+Prepare pull-request for updating `neofs-api-go` library in
```
Comment (cthulhu-rider) [inline docs/release-instruction.md:31]: I suggest to not add it: lower level lib should not trigger the updating of the dependent lib.

File: docs/release-instruction.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Release instructions
+
+## Pre-release checks
+
+These should run successfully:
+* unit-tests,
+* `go mog tidy` (should not change any files);
+* `./prepare.sh /path/to/neofs-api/on/your/machine` (should not change any files).
+
```
Comment (cthulhu-rider) [inline docs/release-instruction.md:9]:
```
* gofmt -w ./
* golangci-lint run ./...
```

File: docs/release-instruction.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Release instructions
+
+## Pre-release checks
+
+These should run successfully:
+* unit-tests,
```
Comment (cthulhu-rider) [inline docs/release-instruction.md:6]:
```suggestion
* `go test ./...`
```

File: docs/release-instruction.md (hunk 1 above)
Comment (carpawell) [reply docs/release-instruction.md:20]: wanted to make my first release and after that updated that instruction

added tag making section

File: docs/release-instruction.md (hunk 3 above)
Comment (alexvanin) [reply docs/release-instruction.md:11]: > is it a good idea to write such instructions without mentioning the version of such utils? it also relates to ./prepare.sh and protoc

This instruction is targeted for repository maintainers, I think they can figure out the version of standard go tools we use. golangci-lint configured with fixed amount of linters as well. As for `prepare.sh`, any protoc version update trigger file changes, so it is not a problem too.

File: docs/release-instruction.md (hunk 2 above)
Comment (alexvanin) [reply docs/release-instruction.md:45]: Yeah, let's drop `Post-release` section in library repository.

File: docs/release-instruction.md (hunk 3 above)
Comment (carpawell) [reply docs/release-instruction.md:11]: ok

File: docs/release-instruction.md (hunk 2 above)
Comment (carpawell) [reply docs/release-instruction.md:45]: dropped

**Review by alexvanin — APPROVED** [review]

**Pull request merged**